ckan/lib/repoze_plugins/auth_tkt.py

```diff
@@ -69,9 +92,15 @@ def make_plugin(secret=None,
     if timeout is not None and reissue_time is None:
         reissue_time = int(math.ceil(int(timeout) * 0.1))
     # Set httponly based on config value. Default is True
-    httponly = config.get(u'who.httponly', True)
+    httponly = _bool(config.get(u'who.httponly', True))
     # Set secure based on config value. Default is False
-    secure = config.get(u'who.secure', False)
+    secure = _bool(config.get(u'who.secure', False))
+    # Set samesite based on config value. Default is lax
```

Comment:
This should be documented as a potentially breaking change (admittedly it *shouldn't* break sensible setups).

Reply:
Changelogs are updated during the release process, so it would be documented there.